apps/src/lab2/views/ExtraLinksModal.tsx

```diff
@@ -427,13 +420,14 @@ const ProjectLinkData: React.FunctionComponent<ProjectLinkDataProps> = ({
                 onBookmark={onBookmark}
               />
             </li>
-            <li>
-              <AbuseScoreInfo
-                abuseScore={abuseScore}
-                onResetAbuseScore={onResetAbuseScore}
-                onReportAbuse={onReportAbuse}
-              />
-            </li>
+            {abuseScore !== undefined && (
```

Comment:
when is the abuse score undefined? Is there ever a case where we would want the ability to report abuse if it's undefined?

Reply:
> when is the abuse score undefined? 
If a user doesn't have project validator permissions, then `projectLinkData` remains `undefined`. https://github.com/code-dot-org/code-dot-org/blob/a81356f259e41237e32ad010077f32791867ad54/apps/src/lab2/views/ExtraLinks.tsx#L50-L57
I defined `abuseScore` as a state value in `ExtraLinksModal` so that if a project validator updates the abuse score, the Extra Links modal re-renders with updated score.